source/Playnite/ViewModels/MainViewModelBase.cs

```diff
@@ -125,12 +125,25 @@ public BaseCollectionView GamesView
 
         public List<FilterPreset> SortedFilterPresets
         {
-            get => Database.FilterPresets.OrderBy(a => a.Name).ToList();
+            
+            get
+            {
+                var sortingOrder = Database.GetFilterPresetsSettings().SortingOrder;
+                return Database.FilterPresets
+                    .OrderBy(i => sortingOrder.IndexOf(i.Id))

```

Comment:
What will this do to existing presets after an update with this change when there's not going to be any sorting data at all?

Reply:
The method returns `-1` if the filter preset Id is not found so those will simply show first in the list until the user manually arranges them if they wish in the Library Manager. I think that's completely fine and has no issues

Reply:
Refactored to use single method and also made new items show last in https://github.com/JosefNemec/Playnite/pull/3258/commits/19bbc7b96e9ec8541f05d8ac4bceea570cd2814c